install_files/securedrop-keyring/DEBIAN/preinst

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Solution adapted from DKG's work on `deb.torproject.org-keyring`.
+# In SecureDrop versions before 0.3.10, the Ansible playbooks used
+# the apt_key module to add the signing key to /etc/apt/trusted.gpg.
+# It's cleaner to use the trusted.gpg.d subdirectory, since we can
+# update that trivially in future versions of the keyring package.
+#
+# Therefore let's clean up prior versions of the key installed
+# to the general apt keyring, to ensure we only have one signing key
+# installed for authenticating securedrop-related packages.
+
+if [ -e /etc/apt/trusted.gpg ] && which gpg >/dev/null; then
+   (
+   h="$(mktemp -d)"
+   trap "rm -rf '$h'" EXIT
+
+   if gpg --homedir="$h" \
+          --batch --no-tty --no-default-keyring --keyring /etc/apt/trusted.gpg \
+          --list-key 0xBFFDF68849BC8719F1BE0191321A7B10AE2C8E02 > /dev/null 2>&1 ; then
+      gpg --homedir="$h" \
+          --batch --no-tty --no-default-keyring --keyring /etc/apt/trusted.gpg \
+          --no-auto-check-trustdb \
+          --delete-key 0xBFFDF68849BC8719F1BE0191321A7B10AE2C8E02 || true
```

Comment:
What key is this fingerprinting referring to? I expected this to be the old (about to be rotated) "Freedom of the Press Foundation Master Signing Key", but it is not:

```
securedrop ~> gpg -k "Freedom of the Press Foundation Master Signing Key"
pub   4096R/0xBADDE0C7FC9F6818 2014-10-26 [expires: 2016-10-27]
      Key fingerprint = B89A 29DB 2128 160B 8E4B  1B4C BADD E0C7 FC9F 6818
uid                 [  full  ] Freedom of the Press Foundation Master Signing Key
sub   4096R/0xE02E0B594833B9A3 2014-10-26
```